docs_app/content/guide/operators.md

```diff
@@ -0,0 +1,219 @@
+# Operators
+
+RxJS is mostly useful for its *operators*, even though the Observable is the foundation. Operators are the essential pieces that allow complex asynchronous code to be easily composed in a declarative manner.
+
+## What are operators?
+
+Operators are **functions** that can be piped to Obseravales, such as `map(...)`, `filter(...)`, `merge(...)`, etc. When called, they do not *change* the existing Observable instance. Instead, they return a *new* Observable, whose subscription logic is based on the first Observable.
+
+<span class="informal">An Operator is a function which creates a new Observable based on the current Observable. This is a pure operation: the previous Observable stays unmodified.</span>
+
+An Operator is essentially a pure function which takes one Observable as input and generates another Observable as output. Subscribing to the output Observable will also subscribe to the input Observable. In the following example, we create a custom operator function that multiplies each value received from the input Observable by 10:
+
+```js
+const multiplyByTen = () => (source) => {
+  return new Observable((subscriber) => {
+    source.subscribe({
+      next(value) { subscriber.next(10 * value); },
+      error(err) { subscriber.error(err); },
+      complete() { subscriber.complete() },
+    });
+  });
+};
+
+const input = from([1, 2, 3, 4]);
+const output = input.pipe(multiplyByTen());
+output.subscribe(x => console.log(x));
+```
+
+Which outputs:
+
+```none
+10
+20
+30
+40
+```
+
+Notice that a subscribe to `output` will cause the `input` Observable to be subscribed. We call this an "operator subscription chain".
+
+You can also create custom operators by wrapping calls to existing operators. We can reimplement our `multiplyByTen` operator by wrapping the `map` operators which will project our multiplicatoin function.
```

Comment:
here's a typo 'multiplicatoin' it should be 'multiplication'

Reply:
should the 'operators' after `map` be singular?